portfolio/src/main/webapp/index.html

```diff
@@ -1,83 +1,99 @@
 <!DOCTYPE html>
 <html>
 
-<head>
-	<meta charset="UTF-8">
-	<title>Yuyan's Portfolio</title>
-	<link rel="stylesheet" href="style.css">
-	<script src="script.js"></script>
-</head>
+  <head>
```

Comment:
You made a white-space only change to this file. Can you revert it? You are perhaps switching between `vi` and the cloud editor and they have different white space conventions?

Reply:
This is interesting. I do not see this white space when I opened index.html in both vim and the editor. What could I do to revertit?